portal-impl/src/com/liferay/portal/service/impl/ImageLocalServiceImpl.java

```diff
@@ -265,6 +330,23 @@ protected void validate(String type) throws PortalException {
 		}
 	}
 
+	private long _getImageCompanyId(long imageId) {
+		Image image = getImage(imageId);
```

Comment:
In the case of the initial logo, there is no image(on DB) so always will be null